agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -772,64 +772,6 @@ bool monitor_thread::handle_cmdu_vs_message(Socket *sd, ieee1905_1::CmduMessageR
         received_error_notification_ack_retry = 0;
         break;
     }
-    case beerocks_message::ACTION_MONITOR_CLIENT_START_MONITORING_REQUEST: {
```

Comment:
This has to be tested on RDKB/UGW - since UTAF is not yet ready I suggest @kantera800 you test it.

Reply:
@tomereli @vitalii-komisarenko 
This PR is currently fails to build on RDKB/UGW platform so i cant test it.

_/home/akanterx/work/dev2/rdkb/atom_rdkbos/build/tmp/work/core2-32-rdk-linux/prplmesh/git+f8786935bdc0b58021710248bf2620896b74c44f-r0/git/agent/src/beerocks/monitor/monitor_thread.cpp:789:66: error: 'beerocks_message::sClientMonitoringParams {aka struct beerocks_message::sClientMonitoringParams}' has no member named 'vap_id'
          int vap_id                       = int(request->params().vap_id);_

Reply:
I was able to build it successfully on RDKB and UGW.